lib/response.js

```diff
@@ -1,11 +1,10 @@
 'use strict';
 
-const _ = require('lodash');
-
 const REQUIRED_PROPERTIES = [
   'url',
   'body',
   'statusCode',
+  'statusText',
```

Comment:
I'm not sure how 'required' these are. Given there's got to be major version bump with this then it might be worth noting any changes and adding them to release notes/GH releases.

Reply:
removed